### ISSUE TITLE
[Request] beekeeper-studio

### ISSUE THREAD

**Issue by Pakrohk** [chaotic-aur/packages#2285]
### Link to the package base(s) in the AUR

https://aur.archlinux.org/packages/beekeeper-studio

### Utility this package has for you

Modern and easy to use SQL client for MySQL, Postgres, SQLite, SQL Server, and more

### Do you consider the package(s) to be useful for every Chaotic-AUR user?

No, but for a few.

### Do you consider the package to be useful for feature testing/preview?

- [ ] Yes

### Have you tested if the package builds in a clean chroot?

- [X] Yes

### Does the package's license allow redistributing it?

YES!

### Have you searched the issues to ensure this request is unique?

- [X] YES!

### Have you read the README to ensure this package is not banned?

- [X] YES!

**Comment by Technetium1** [comment]
Thanks for requesting this! I have been waiting to add this pending a version of Electron that is not [end-of-life](https://endoflife.date/electron); as it's currently **WAY** back on version 13.

**Comment by Technetium1** [comment]
They're now up to Electron 18—which is still EoL, but we're finally getting closer!

**Comment by xiota** [comment]
Went ahead and added this because all versions of electron are EOL.

To see which packages need which version:
```
for i in 16 17 18 19 21 ; do
   echo
   comm -13 <(pactree -usr electron | sort -u) <(pactree -usr electron${i}-bin | sort -u)
done
```

**Comment by xiota** [comment]
The [build log](https://builds.garudalinux.org/repos/chaotic-aur/logs/#beekeeper) is 1 GB.  Any way to clear unnecessary logs?

**Comment by Technetium1** [comment]
I don't think you can remove them individually. Luckily, there is still plenty of free space.

**Comment by xiota** [comment]
This package was very resource intensive to build.  Would like to drop when the current build goes out of date.  Would switching to `beekeeper-bin` be acceptable?

**Comment by Technetium1** [comment]
@xiota yes it would be since they've revised `-bin` :)

**Comment by dpeukert** [comment]
Not sure if it's any help with the resource usage in general, but the issue that required the debug logs to be kept on ([see AUR comment](https://aur.archlinux.org/packages/beekeeper-studio#comment-934477)) was fixed upstream at some point, so the build log size is back to reasonable levels.

**Comment by xiota** [comment]
@dpeukert Thank you for the notification.  Unfortunately, cannot re-add until project updates to a supported Electron version.  (Currently, Electron 27 or newer.)  Appears upstream would require some significant changes to update to Electron 19+.

**Comment by xiota** [comment]
Closing because upstream doesn't keep Electron up to date.  Currently uses Electron 31.7.3, which was EOL Jan 2025.
